ui/panels-plugin/src/plugins/gauge-chart/GaugeChartOptionsEditorSettings.test.tsx

```diff
@@ -66,7 +66,7 @@ describe('GaugeChartOptionsEditorSettings', () => {
     const calcSelector = screen.getByRole('combobox', { name: 'Calculation' });
     userEvent.click(calcSelector);
     const meanOption = screen.getByRole('option', {
-      name: 'Mean',
+      name: /Mean/,
```

Comment:
Switching these selectors to regex because the content of the `option` also includes descriptions now, and it would be annoying to test against the full thing, especially because descriptions may be more likely to be changed.